core/trino-main/src/main/java/io/trino/execution/buffer/OutputBufferMemoryManager.java

```diff
@@ -66,6 +76,7 @@ public OutputBufferMemoryManager(long maxBufferedBytes, Supplier<LocalMemoryCont
         this.maxBufferedBytes = maxBufferedBytes;
         this.memoryContextSupplier = Suppliers.memoize(memoryContextSupplier::get);
         this.notificationExecutor = requireNonNull(notificationExecutor, "notificationExecutor is null");
+        this.lastBufferUtilizationRecordTime = ticker.read();
     }
 
     public void updateMemoryUsage(long bytesAdded)
```

Comment:
Could you check how much overhead updating of `outputBufferUtilization` adds? We do that under sync-lock, so it shouldn't be too long

Reply:
I run concurrency-benchmarks and I do not see a higher lock congestion at worker side. I wait for results from "serial" benchmarks but I do not expect performance impact.

Reply:
Duration TPCH: -1.00 
Duration TPCDS: -1.54 
CPU TPCH 0.47
CPUT TPCDS 0.87

I do not see `TDigest` and `TDistribution` stuff in the JFR profile.